docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# vim: set ft=dockerfile:
+FROM ruby:2.6-slim
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    DB_HOST="postgres" \
+    DB_NAME="chatwoot" \
+    DB_USER="chatwoot" \
+    DB_PASS="" \
+    RAILS_ENV="production" \
+    REDIS_URL="redis://redis:6379" \
+    SECRET_KEY_BASE="lithahgaef6aSeGhoh8Aelei0Aemoh6geequeec1oerohLichio1bie3eeYah5oh"
+
+RUN apt-get update \
+    && apt-get -qq -y install \
+    build-essential \
+    curl \
+    git \
+    imagemagick \
+    libpq-dev \
+    && curl -L https://deb.nodesource.com/setup_12.x | bash - \
+    && curl https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo 'deb https://dl.yarnpkg.com/debian stable main' > /etc/apt/sources.list.d/yarn.list \
+    && apt-get update \
+    && apt-get -qq -y install nodejs yarn \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt
+
+RUN git clone https://github.com/chatwoot/chatwoot
```

Comment:
shouldn't we be using the local version of the code instead of pulling it again from the repo ?

Reply:
sure, I just started this entirely separetely